### PR TITLE
Pushing clean scoring script and evaluation notebook

### PR DESCRIPTION
**Pull request by nthain**
- Adds biasbios and scrubbed biasbios datasets to input_fn_example.py.
- Add scoring script score_test_data.py which can be used to score deployed models on available datasets. This needs to be expanded to accomodate other datasets which is left as a TODO for later.
- Adds helper bash scripts score_bias_data.sh and score_scrubbed_data.sh for the biasbios research project.
- Adds evaluation notebook for biasbios research.
- Minor changes to scripts in the utils_export folder to accomodate the multiclass setting.

### CODE REVIEW

File: model_evaluation/input_fn_example.py [hunk 1]
```diff
@@ -275,3 +275,146 @@ def input_fn_bias(max_n_examples):
...
+    res = res.rename(columns={'raw_text': model_input_comment_field})
+    return res
+
+  return input_fn_bias
+
+#### #### #### #### #### #### #### ####
+####   SCRUBBED BIASBIOS DATASET   ####
+#### #### #### #### #### #### #### ####
+
+SCRUBBED_BIASBIOS_PATH = 'gs://conversationai-models/biosbias/dataflow_dir/data-preparation-20190225173815_scrubbed/test-00000-of-00003.tfrecord'
```
Comment (fprost) [inline model_evaluation/input_fn_example.py]: I am not a fan of creating an entire new function "create_input_fn_scrubbed_biasbios" where it is identical than the non-scrubbed one. I would prefer the path to be a parameter.

File: model_evaluation/utils_export/dataset.py [hunk 2]
```diff
@@ -265,7 +266,7 @@ def wait_predictions(self, model):
     for job_id in model.job_ids_prediction():
       utils_cloudml.check_job_over(model.project_name(), job_id)
 
-  def add_model_prediction_to_data(self, model, recompute_predictions=True):
+  def add_model_prediction_to_data(self, model, recompute_predictions=True, class_names = None):
```
Comment (fprost) [inline model_evaluation/utils_export/dataset.py]: not: no space.

File: model_evaluation/Eval Notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,2082 @@
+{
```
Comment (fprost) [inline model_evaluation/Eval Notebook.ipynb]: Can you rename file to contains "biosbias"?

File: model_evaluation/input_fn_example.py (hunk 1 above)
Comment (nthain) [reply model_evaluation/input_fn_example.py]: Done. PTAL to make sure the fix looks right.

File: model_evaluation/utils_export/dataset.py (hunk 2 above)
Comment (nthain) [reply model_evaluation/utils_export/dataset.py]: Done.

File: model_evaluation/Eval Notebook.ipynb (hunk 3 above)
Comment (nthain) [reply model_evaluation/Eval Notebook.ipynb]: Done.

**Comment by nthain** [comment]
Made the fixes you suggested. Thanks for the review. PTAL.

To answer your other two questions:

> Do we want to push in a separate branch than master? Not sure if you can push in the branch in my fork.
I think it's ok to push it to master temporarily until we refactor the research work into it's own directory.

> I am not clear how you generate the 2 csv before (standard_data_path). I guess you adapted and run the other notebook?
These were generated using the 'score_bias_data.sh' and 'score_scrubbed_data.sh' scripts included in the PR. Is this what you mean?

**Comment by fprost** [comment]
I missed the score_bias_data.sh. Super useful script. Thanks a lot Nithum!

**Pull request merged**